lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions_controller.rb

```diff
@@ -27,10 +27,25 @@ def review
     @content_block_edition = ContentObjectStore::ContentBlock::Edition.find(params[:id])
   end
 
+  EDIT_FORM_STEPS = {
+    edit_block: "edit_block",
+    review_links: "review_links",
+  }.freeze
+
   def edit
+    step = params[:step]
+
+    case step
+    when EDIT_FORM_STEPS[:edit_block]
```

Comment:
could the steps or logic be abstracted to a helper?

Reply:
We could try to move the steps to a helper. That will be available to controllers, views and can be included into classes like the form objects. I'm not sure about view components.

```ruby
edit_form_step_for(params[:edit_block])
```

It would also mean we don't have to use the long module name which I can't think around that. We might be able to assign it to a shorter constant in our engine initialiser?

```ruby 
EDIT_FORM_STEPS = ContentObjectStore::ContentBlock::EditionsController::EDIT_FORM_STEPS
```

Reply:
> We could try to move the steps to a helper. That will be available to controllers, views and can be included into classes like the form objects. I'm not sure about view components.

Shall I leave further refactoring - e.g. to a helper - to when we introduce a third step, to keep this PR smaller and get it over the line?

> We might be able to assign it to a shorter constant in our engine initialiser?

will take a look at this!

Reply:
> We might be able to assign it to a shorter constant in our engine initialiser?

Tried doing this (also suggested in the engines docs, [to pub configuration in the module](https://guides.rubyonrails.org/engines.html)) but with no success as still didn't seem to recognise it in other files :( 
I will leave as is - I think referencing with long module name is still better than having random strings floating about.